doc/guide.md

```diff
@@ -3103,7 +3103,8 @@ entry. For example:
           "methods": [ "POST" ],
           "pathPattern": "/testb/1",
           "schedule": {
-            "cron": "0 6,18 * * *"
+            "cron": "0 6,18 * * *",
+            "zone": "CET"
```

Comment:
Not sure if we need a link for valid zoneId. For example https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/ZoneId.html

Reply:
good idea. I have added something